website/docs/r/bfd_multihop_interface_profile.html.markdown

```diff
@@ -0,0 +1,58 @@
+---
+subcategory: "L3Out"
+layout: "aci"
+page_title: "ACI: aci_bfd_multihop_interface_profile"
+sidebar_current: "docs-aci-resource-bfd_multihop_interface_profile"
+description: |-
+  Manages ACI BFD Multihop Interface Profile
+---
+
+# aci_bfd_multihop_interface_profile #
+
+Manages ACI BFD Multihop Interface Profile
+
+## API Information ##
+
+* `Class` - bfdMhIfP
+* `Distinguished Name` - uni/tn-{tn_name}/out-{l3out_name}/lnodep-{ln_name}/lifp-{lifp_name}/bfdMhIfP
+
+## GUI Information ##
+
+* `Location` -  Tenant -> Networking -> L3Out -> Logical Node Profiles -> Logical Interface Profiles 
+
+
+## Example Usage ##
+
+```hcl
+resource "aci_bfd_multihop_interface_profile" "example" {
+  logical_interface_profile_dn = aci_logical_interface_profile.example.id
+  annotation                   = "orchestrator: terraform"
+  key                          = "SomeAuthKey"
+  key_id                       = "1"
+  interface_profile_type       = "none"
+  bfd_rs_mh_if_pol             = aci_resource.example.id
+}
+```
+
+## Argument Reference ##
+
+* `logical_interface_profile_dn` - (Required) Distinguished name of the parent BFD Multihop Logical Interface Profile object.
+* `annotation`                   - (Optional) Annotation of the BFD Multihop Interface Profile object.
+* `key`                          - (Optional) Authentication key.
+* `key_id`                       - (Optional) Authentication Key ID. Allowed range is 1-255 and default value is "1". Type: String.
```

Comment:
key lower case in on and upper case in the other, is this correct?

Reply:
Key is made upper case everywhere.